src/cuda/1d/interp1d_wrapper.cu

```diff
@@ -14,6 +14,52 @@ using namespace cufinufft::memtransfer;
 namespace cufinufft {
 namespace spreadinterp {
 
+template <typename T>
```

Comment:
I'm confused where this code came from (surely it already exists in cufinufft?)

Reply:
It did indeed, it was however removed during cleanup as a part of https://github.com/flatironinstitute/finufft/commit/27c3621b8561f10ad94a8eb45695fd4c533181bc

This was perhaps done as it was used to test internal modules, but then it was this very functions which I chose to expose (with a wrapper)